save-cloud-common/src/commonMain/kotlin/org/cqfn/save/execution/ExecutionUpdateCreationDto.kt

```diff
@@ -0,0 +1,20 @@
+package org.cqfn.save.execution
+
+import org.cqfn.save.entities.Project
+import kotlinx.serialization.Serializable
+
+/**
+ * @property project
+ * @property testSuiteIds
+ * @property resourcesRootPath
+ * @property batchSize
+ * @property version
+ */
+@Serializable
+data class ExecutionUpdateCreationDto(
```

Comment:
```suggestion
data class ExecutionInitializationDto(
```
or someting similar? Execution is already created by the time this class is used